src/js/net.js

```diff
@@ -224,6 +238,11 @@ Socket.prototype.setKeepAlive = function(enable, delay) {
 };
 
 
+Socket.prototype.address = function() {
+  return this._getsockname();
+};
```

Comment:
How do you think rename function name ``_getsockname()`` to ``address``, because it looks just one line wrapper if there isn't any issue?